### ISSUE TITLE
Fixture podspecs are manually duplicated from cocoapods-specs

### ISSUE THREAD

**Issue by nolanw** [CocoaPods/CocoaPods#32]
The podspecs bundled as fixtures in `spec/fixtures/spec-repos/master.tar.gz` need to be manually updated after pulling from alloy/cocoapods-specs. It makes more sense to me to include cocoapods-specs as a submodule so it's easier to test podspecs as they are in production. However, I got stuck on two issues when I tried to do this myself:
- The functional tests become a moving target as the search results will change with specs being added.
- Including an old podspec (e.g. ASIHTTPRequest 1.8) alongside a newer one (1.8.1 is the only one currently there) for testing purposes becomes nontrivial.

The other problem here is someone will still have to remember to keep the cocoapods-specs submodule moving along. So I'm thinking there's a better solution out there.

tl;dr it would be nice to test with production podspecs but in a more controlled environment.
